src/persist/src/operators/stream.rs

```diff
@@ -649,7 +649,10 @@ pub trait RetractFutureUpdates<G: Scope<Timestamp = u64>, K: TimelyData, V: Time
         name: &str,
         write: StreamWriteHandle<K, V>,
         upper_ts: u64,
-    ) -> Stream<G, ((K, V), u64, isize)>;
+    ) -> (
```

Comment:
not this line but I think the comments on line 645 are now out of date?

Reply:
Fixing!